recipe/meta.yaml

```diff
@@ -1,40 +1,43 @@
 {% set name = "ipython_memory_usage" %}
-{% set version = "1.1" %}
+{% set version = "1.2" %}
+{% set sha256 = "1f4697210257f853fea74de0cf3fae60a32e550e578bac6f46de9b40c550422b" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
+  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 1f4697210257f853fea74de0cf3fae60a32e550e578bac6f46de9b40c550422b
+  sha256: {{ sha256 }}
 
 build:
   noarch: python
   number: 0
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
 
 requirements:
-  host:
+  build:
     - python
     - pip
+
   run:
     - python
-    - memory_profiler
     - ipython
 
 test:
   imports:
-    - ipython_memory_usage
+   - ipython_memory_usage
 
 about:
   home: https://github.com/ianozsvald/ipython_memory_usage
   license: BSD-2-Clause
   license_family: BSD
-  license_file: {{ RECIPE_DIR }}/LICENSE
-  summary: 'IPython tool to report memory usage deltas for every command you type'
-  dev_url: https://github.com/ianozsvald/ipython_memory_usage
+  license_file: LICENSE
+  summary:
+    - IPython tool to report memory usage deltas for every command you type. If you are running out of RAM then use this tool to understand what’s happening. It also records the time spent running each command.
```

Comment:
I think this needs to be a single string

Reply:
This seems ok as is.

Reply:
This is still a list, not a string